libcxx/include/__ranges/join_view.h

```diff
@@ -234,8 +243,37 @@ namespace ranges {
         __inner_.reset();
     }
 
+    _LIBCPP_HIDE_FROM_ABI constexpr _Outer& __get_outer() {
+      if constexpr (forward_range<_Base>) {
+        return __outer_;
+      } else {
+        return *__parent_->__outer_;
+      }
+    }
+
+    _LIBCPP_HIDE_FROM_ABI constexpr const _Outer& __get_outer() const {
+      if constexpr (forward_range<_Base>) {
+        return __outer_;
+      } else {
+        return *__parent_->__outer_;
+      }
+    }
+
+    _LIBCPP_HIDE_FROM_ABI constexpr __iterator(_Parent& __parent, _Outer __outer)
```

Comment:
Can you please add tests for these constructors? You removed the test in `libcxx/test/std/ranges/range.adaptors/range.join.view/iterator/ctor.parent.outer.pass.cpp` but I don't think you added any for the new ctors. Note that those are exposition only ctors so the tests could go in `libcxx/test/libcxx/<...>` instead of `libcxx/test/std/<...>` -- or if you prefer you can put them in `libcxx/test/std` but mark them with `REQUIRES: stdlib=libc++`.

Reply:
I don't think this has been addressed (unless I missed it)?